scripts/visualise_results.py

```diff
@@ -34,16 +35,21 @@ def plot_data(df, number_of_days_to_plot):
 
     folders = sorted(glob(f"{parsed_variables['output_dir']}/*"))
     results = get_results(folders, number_of_days)
+
     result_fields = [
         [
             result[0],
-            result[1].average_weighted_get,
-            result[1].average_weighted_post,
-            result[1].average_weighted_total,
+            *(
+                result[1].percentiles[percentile]
+                for percentile in BenchmarkStats.PERCENTILES_TO_GRAPH
+            ),
         ]
         for result in results
     ]
 
-    data_frame = DataFrame(result_fields, columns=["DATE", "GET", "POST", "AVERAGE"])
+    percentile_columns = (
+        f"{percentile}th" for percentile in BenchmarkStats.PERCENTILES_TO_GRAPH
```

Comment:
I think the value for `PERCENTILES_TO_GRAPH` should be defined in this file, as it is only used in this file.